core/spec/models/spree/image_spec.rb

```diff
@@ -9,6 +9,26 @@
     let(:default_style) { :product }
   end
 
+  describe 'validations' do
```

Comment:
I feel like there's no need for the `describe` blocks here since we don't have any shared object between the test cases. What about just using two different tests as:

```ruby

it 'is valid when attachment has allowed content type' do
  image = build(:image, 
    attachment: File.open(File.join('lib', 'spree', 'testing_support', 'fixtures', 'blank.jpg'))
  )
  
  expect(image).to be_valid
end

it 'is not valid when attachment has restricted content type' do
  image = build(:image, 
    attachment: File.open(File.join('lib', 'spree', 'testing_support', 'fixtures', 'file.txt'))
  )
  
  expect(image).to_not be_valid
end
```

Reply:
This syntax has been implemented!